pom.xml

```diff
@@ -82,5 +94,16 @@
             <artifactId>SimpleNLG-DE</artifactId>
             <version>1.1</version>
         </dependency>
+
+        <dependency>
```

Comment:
Wofür brauchen wir/was ist slf4j? Die Imports kommen doch ausschließlich von apache.logging.log4j

Reply:
Das brauchen wir, weil sonst beim Template Resolver durch Apache Velocity eine Warnung kommt, dass slf4j nicht gefunden wird. Die wurde beim Starten immer in rot angezeigt.